backend/backend/roll_die.py

```diff
@@ -66,6 +67,7 @@ def player_roll_dice(source=sys.stdin, output=sys.stdout):
                     player.balance += pass_go_amount
                     player.board_position -= number_of_squares
 
-                # check_position(player_id)
+                # card_details = check_position(player_id)
```

Comment:
Is this meant to be commented? On my screen it is, but I think it possibly shouldn’t be.

Reply:
This function was causing database errors earlier when Gary was testing it. It seemed to have been related to the properties table in the database which is probably fixed now. On that note, it's probably worth uncommenting. I'll push this change momentarily